python/feathub/feature_views/feature_view.py

```diff
@@ -44,6 +46,10 @@ def __init__(
                          which refers to a feature in the source table.
         :param keep_source_fields: True iff all fields in the source table should be
                                    included in this table.
+        :param timestamp_field: Optional. If it is not None, it is the name of the field
```

Comment:
The semantics of the `timestamp_field` in `TableDescriptor` is different from the `timestamp_field` here. For `TableDescriptor`, if it is None, then the TableDescriptor won't have any field that represents when the row is generated. For `FeatureView`, if it is None, then we will still try to re-use the `timestamp_field` from the source table.

Thus we can not simply re-use the doc from `TableDescriptor`.

How about this:

Optional. If not None, the feature with the given name is used as the timestamp_field of the TableDescriptor represented by this FeatureView. Otherwise, the timestamp_field of the source TableDescriptor is used as the timestamp_field of the TableDescriptor represented by this FeatureView.